### PR TITLE
add Makefile

### PR DESCRIPTION
**Pull request by bvered**
Signed-off-by: 662962756 <bvered@il.ibm.com>

### CODE REVIEW

**Comment by bvered** [comment]
[create MakeFile for console - to build and push images](https://trello.com/c/nRvuF1Ia/107-create-makefile-for-console-to-build-and-push-images)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+IMAGE_REPO="docker.io"
+NAME_SPACE="ibmcom"
+CONSOLE_IMAGE_VERSION=1.3.0
+
+CONSOLE_NAME=ibm-storage-odf-plugin
+
+CONSOLE_IMAGE=$(IMAGE_REPO)/${NAME_SPACE}/$(CONSOLE_NAME)
```
Comment (tyichye19) [inline Makefile:7]: Can we add the `CONSOLE_IMAGE` in here, instead of adding it to every docker command?

Comment (bvered) [reply Makefile]: fixed

**Review by tyichye19 — APPROVED** [review]
LGTM

**Pull request merged**